package.json

```diff
@@ -56,6 +56,7 @@
     "rollup": "^1.31.0"
   },
   "dependencies": {
-    "expr-eval": "^2.0.2"
+    "expr-eval": "^2.0.2",
+    "typescript": "^4.5.5"
```

Comment:
Is there any reason for this to be an install/runtime dependency? Seems like it should be only a devDependency.

Reply:
Yes this was an oversight on my part. Just published v0.3.8 with typescript moved to dev dependencies. Thanks!

Reply:
Awesome, thank you!